src/pages/AddSeparators.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <keep-alive>
-    <add-separator-form
-      :someShelf="computedUpdatingShelf"
-    ></add-separator-form>
+    <add-separator-form :someShelf="computedUpdatingShelf"></add-separator-form>
```

Comment:
I'm not sure because it's been a long time in vue but I think it's like that:
```suggestion
    <add-separator-form :some-shelf="computedUpdatingShelf"></add-separator-form>
```